[docs] CDCSDK: Modify section to mention steps to DROP/TRUNCATE tables with CDC (#16241)
yugabyte/yugabyte-db: docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md

When a row is deleted, the _delete_ event value still works with log compaction, because Kafka can remove all earlier messages that have that same key. However, for Kafka to remove all messages that have that same key, the message value must be `null`. To make this possible, the YugabyteDB connector follows a delete event with a special _tombstone_ event that has the same key but a null value.

> {{< tip title="Enable GFlag to DROP or TRUNCATE" >}}

Reviewer: ```suggestion
{{< tip title="DROP or TRUNCATE tables when CDC is enabled" >}}
```

> The YugabyteDB CDC implementation will not let you DROP or TRUNCATE a table while an active CDC stream is present on the namespace. If you need to perform these operations while CDC is enabled, set the value of a GFlag `enable_delete_truncate_cdcsdk_table` to `true` and then you can DROP or TRUNCATE the table.

Reviewer: ```suggestion
By default, the YugabyteDB CDC implementation does not allow you to DROP or TRUNCATE a table while an active CDC stream is present on the namespace. To perform these operations while CDC is enabled, set the [enable_delete_truncate_cdcsdk_table](../../../reference/configuration/yb-tserver/#enable-delete-truncate-cdcsdk-table) flag to `true`.
```
